raiden-dapp/src/router/index.ts

```diff
@@ -70,6 +70,40 @@ const router = new Router({
         title: 'Channels'
       },
       component: () => import('../views/Channels.vue')
+    },
+    {
+      path: '/general',
+      name: RouteNames.GENERAL,
+      beforeEnter: (to, from, next) => {
+        if (from.name === null) {
```

Comment:
We have already clarified this, but some comments for others to quickly understand what is done here might be beneficial